### DISCUSSION TITLE
2023년 10월 15일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#88] [General]
# 2023년 10월 15일

TODO 글쓰기 미션  [priority:: medium]  [due:: 2023-10-30] 바론 코드리뷰  [priority:: highest]  [created:: 2023-10-15]  [scheduled:: 2023-10-15]  [completion:: 2023-10-15] 스터디룸 예약  [priority:: highest]  [due:: 2…

https://vero.wiki/2023-10-15/

**Comment by somsom13** [comment]
제가 베로의 우선순위 1등이 되다니.. 너무 두근거려요

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 이 여성.. 제법 웃기다...
